Subject: Search for fonts generated by mknfonts, Debian-specific

* Source/art/FTFontEnumerator.m : Add /var/lib/GNUstep to the search path

Forwarded: not-needed
From: Hubert Chathi <uhoreg@debian.org>
From: Yavor Doganov <yavor@gnu.org>
Last-Update: 2016-07-04

--- a/Source/art/FTFontEnumerator.m
+++ b/Source/art/FTFontEnumerator.m
@@ -349,7 +349,11 @@
   families_seen = [[NSMutableSet alloc] init];
   families_pending = [[NSMutableSet alloc] init];
 
-  paths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSAllDomainsMask, YES);
+  /* Search for fonts in all domains including those generated by
+     mknfonts in postinst.  */
+  paths = [NSSearchPathForDirectoriesInDomains (NSLibraryDirectory,
+						NSAllDomainsMask, YES)
+	      arrayByAddingObject: @"/var/lib/GNUstep"];
   for (i = 0; i < [paths count]; i++)
     {
       path = [paths objectAtIndex: i];
